examples/text-generation/utils.py

```diff
@@ -102,7 +102,7 @@ def setup_inference(args, model):
     print("Initializing inference mode")
     const_marking = os.getenv("ENABLE_CONST_MARKING", "True")
```

Comment:
@Yantom1 you should remove it. we should not use this variable anymore. we should always call hpu_initialize in this function. as QA set this flag to false, did you test it correctly?